flutter_src/lib/logic/api/firestore_db.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:macrobaseapp/logic/state/macro_state/macro_notifier.dart';
+import 'package:macrobaseapp/model/adapters/macro_model.dart';
+import 'package:macrobaseapp/model/entities/macro.dart';
+
+class FirestoreService {
```

Comment:
Uhhh... not sure if there is an option to do in GitHub, but it would be much easier to understand if you changed the original file with this logic and not delete old/add new.